File: examples/blink-io8.cpp
#include "stdio.h"

#include "maa.h"

int
main(int argc, char **argv)
{
    fprintf(stdout, "MAA Version: %d\n Starting Blinking on IO8", get_version());
    gpio_t gpio;
    gpio_init(&gpio, 26);
    gpio_dir(&gpio, "out");

    while(1){
        gpio_write(&gpio, 0);
        sleep(1);
        gpio_write(&gpio, 1);
        sleep(1);
    }
    return 0;
}

File: examples/python/blink-io8.py
#!/usr/bin/env python3

import pymaa as maa
import time

x = maa.gpio_t()
maa.gpio_init(x, 26)
maa.gpio_dir(x, "out")

while True:
    maa.gpio_write(x,1)
    time.sleep(0.2)
    maa.gpio_write(x,0)
    time.sleep(0.2)
